plugin-management-cli/src/test/java/io/jenkins/tools/pluginmanager/cli/CliOptionsTest.java

```diff
@@ -216,7 +216,6 @@ public void setupUpdateCenterCliTest() throws Exception {
         assertThat(cfg.getJenkinsIncrementalsRepoMirror()).hasToString(incrementalsCli);
         assertThat(cfg.getJenkinsPluginInfo()).hasToString(pluginInfoCli);
     }
-
```

Comment:
I would prefer that you not remove the empty line between tests.  I makes it more difficult for me to read.  In this case, it seems that is the only change in this file in this pull request, so it should be removed from the pull request.

Reply:
Done @MarkEWaite